pkg/config/manager.go

```diff
@@ -140,6 +142,42 @@ func (m *Manager) All() (configs []*Config) {
 	return
 }
 
+// Attach sets a given config as the current one. This is done by adding an `attached`
+// file next to it. If another config is already attached, the file gets moved.
+func (m *Manager) Attach(config *Config) error {
+	var currentAttachedFile string
+	for _, c := range m.All() {
+		attachedFile := m.attachedFilePath(c)
+		if m.fileExists(attachedFile) {
+			currentAttachedFile = attachedFile
```

Comment:
I've ignored this case for simplicity, do you suggest we handle it? How?

Reply:
In `Current`, an error is triggered because in case of multiple attached files the current cluster can't be determined. 

Here, in case there are multiple attached files, it is theoretically still possible to proceed and meet this function's intent (attach to a given cluster). So it seems unappropriate to return an error in such a case. We should either assume there is only one attached file (if it's not the case then the error is delayed until `Current` is called) or try to be smart and do whatever is necessary to attach to the requested cluster.